admin/addField.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import * as firebase from 'firebase/app';
+
+export const AddField = () => {
+    const [field, setField] = useState('');
+
+    return <form onSubmit={e => addFieldToLinksInDatabase(e, field)}>
+        <label><h3>New field title</h3></label>
```

Comment:
Når du bruker `<label />` så vil du at man skal få fokus på riktig felt når man trykker på label-teksten.

Det er det to måter å gjøre på:

1. wrap `<label>` rundt input-feltet. Det kan være litt tricky å style, men det funker i alle fall bra
2. gi `<input />` feltet ditt en unik ID, og referer til den IDen med `<label htmlFor="IDen-til-input-feltet" />`

Reply:
Det fungerer å gi fokus til feltet ved å trykke på labelen sånn som det er nå.

Reply:
Men det er kanskje bare i Chrome?

Reply:
Merkelig. Det kan hende det er Chrome som er litt snill der ja. Det vil i alle fall feile i en UU-audit